avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -936,16 +948,25 @@ private static Object maybeWrapSchemaless(Schema schema, Object value, String ty
                              JsonNodeFactory.instance.numberNode(schema.version()));
         }
         if (schema.parameters() != null) {
-          JsonNode params = parametersFromConnect(schema.parameters());
-          if (!params.isEmpty()) {
-            baseSchema.addProp(CONNECT_PARAMETERS_PROP, params);
+          if (discardTypeDocDefault) {
+            baseSchema.addProp(CONNECT_PARAMETERS_PROP, parametersFromConnect(schema.parameters()));
+          } else {
+            JsonNode params = parametersFromConnect(schema.parameters());
+            if (!params.isEmpty()) {
+              baseSchema.addProp(CONNECT_PARAMETERS_PROP, params);
+            }
```

Comment:
I'm not sure we need this change.  It seems fine to only add the parameters if they're not empty.

Reply:
@xiaoyali97 , did you get a chance to check if we need this?

Reply:
Sorry I missed this command. Yes you're right, the `discardTypeDocDefault` check is not needed since it will be handled in `parametersFromConnect` already.